src/SayMoreTests/ApplicationContainerTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Linq;
+using NUnit.Framework;
+using SayMore;
+using SayMore.Model.Files;
+using SayMore.Properties;
+
+namespace SayMoreTests
+{
+	[TestFixture]
+	public class ApplicationContainerTests
+	{
+		[Test]
```

Comment:
Could probably use TestCase and go ahead and test all relevant ComponentRoles that follow this pattern.

Reply:
Done